.github/CODEOWNERS

```diff
@@ -6,6 +6,9 @@
 
 # ==== Ray core ====
 
+# API compatibility
+/dashboard/modules/snapshot @wuisawesome
```

Comment:
I don't think this actually works right now, but in theory it should prevent anyone from breaking this API.